Sources/BraintreePayPal/BTPayPalVaultEditRequest.swift

```diff
@@ -7,11 +7,11 @@ public struct BTPayPalVaultEditRequest {
     private let editPayPalVaultID: String
     private let merchantAccountID: String?
 
-    //   TODO: specify endpoint for merchant to retrieve the token
+    //   TODO: specify endpoint for merchant to retrieve the EditResult
```

Comment:
Can we address these TODOs in this PR? The endpoints are server side customer or payment method find.

Reply:
Do we mean where to get the editPayPalVaultID or where to retrieve the updated FI?

Reply:
Both fetching the `editPayPalVaultID` and retrieving the updated FI are the same server side endpoint (customer or payment method find)